File: IAIso-v5.0/core/iaiso-php/src/Coordination/CoordinatorLifecycle.php
<?php

declare(strict_types=1);

namespace IAIso\Coordination;

/** Lifecycle state of a fleet-aggregated coordinator. */
enum CoordinatorLifecycle: string
{
    case Nominal    = 'nominal';
    case Escalated  = 'escalated';
    case Released   = 'released';
}

File: IAIso-v5.0/core/iaiso-java/iaiso-coordination/src/main/java/io/iaiso/coordination/CoordinatorLifecycle.java
package io.iaiso.coordination;

/** Lifecycle state of a fleet-aggregated coordinator. */
public enum CoordinatorLifecycle {
    NOMINAL("nominal"),
    ESCALATED("escalated"),
    RELEASED("released");

    private final String wireValue;
    CoordinatorLifecycle(String w) { this.wireValue = w; }
    @Override public String toString() { return wireValue; }
}
